designer/server/plugins/routes/app.ts

```diff
@@ -39,6 +40,43 @@ export const getAppChildRoutes: ServerRoute = {
   },
 };
 
+export const getErrorCrashReport: ServerRoute = {
+  method: "get",
+  path: "/error/crashreport/{id}",
```

Comment:
I have a minor concern here that anyone can simply pass a form ID into this to get a crash report, but depending on what those reports contain it might be irrelevant anyway.... just noting it down so I can at least consider it..